spec/fixtures/odk/responses/nested_group_form_response_duplicate.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' ?>
```

Comment:
ah good idea to create a lengthy one instead of using sleep to make it take longer, better to match real conditions!